src/features/user/MFV2/ContributionsMap/Markers/CustomMarkers.tsx

```diff
@@ -0,0 +1,125 @@
+import Mangroves from '../../../../../../public/assets/images/icons/myForestV2Icons/Mangroves';
+import ManagedRegeneration from '../../../../../../public/assets/images/icons/myForestV2Icons/ManagedRegeneration';
+import Agroforestry from '../../../../../../public/assets/images/icons/myForestV2Icons/Agroforestry';
+import UrbanRestoration from '../../../../../../public/assets/images/icons/myForestV2Icons/UrbanRestoration';
+import Conservation from '../../../../../../public/assets/images/icons/myForestV2Icons/Conservation';
+import TreePlanting from '../../../../../../public/assets/images/icons/myForestV2Icons/TreePlanting';
+import OtherPlanting from '../../../../../../public/assets/images/icons/myForestV2Icons/OtherPlanting';
+import { contributionLocation } from '../../../../../utils/myForestV2Utils';
+import themeProperties from '../../../../../theme/themeProperties';
+
+interface ProjectTypeIconProps {
+  projectType: string;
+  classification: string;
+}
+
+const ProjectTypeIcon = ({
+  projectType,
+  classification,
+}: ProjectTypeIconProps) => {
+  const chooseColorForMarkerIcon = (classification: string) => {
+    switch (classification) {
+      case 'treePlantation':
+        return `${themeProperties.primaryDarkColorX}`;
+      case 'conservation':
+        return `${themeProperties.mediumBlue}`;
+      case 'restoration':
+        return `${themeProperties.electricPurple}`;
+      default:
+        return null;
+    }
+  };
+
+  switch (projectType) {
+    case 'natural-regeneration':
+      return <NaturalRegeneration />;
+    case 'mangroves':
+      return (
+        <Mangroves
+          width={68}
+          height={75}
+          color={chooseColorForMarkerIcon(classification)}
+        />
+      );
+    case 'managed-regeneration':
+      return (
+        <ManagedRegeneration
+          width={68}
+          height={75}
+          color={chooseColorForMarkerIcon(classification)}
+        />
+      );
+    case 'agroforestry':
+      return (
+        <Agroforestry
+          width={68}
+          height={75}
+          color={chooseColorForMarkerIcon(classification)}
+        />
+      );
+    case 'urban-planting':
+      return (
+        <UrbanRestoration
+          width={68}
+          height={75}
+          color={chooseColorForMarkerIcon(classification)}
+        />
+      );
+    case 'conservation':
+      return (
+        <Conservation
+          width={68}
+          height={75}
+          color={chooseColorForMarkerIcon(classification)}
+        />
+      );
+    case 'large-scale-planting':
+      return (
+        <TreePlanting
+          width={68}
+          height={75}
+          color={chooseColorForMarkerIcon(classification)}
+        />
+      );
+    case 'other-planting':
+      return (
+        <OtherPlanting
+          width={68}
+          height={75}
+          color={chooseColorForMarkerIcon(classification)}
+        />
+      );
+    default:
+      return null;
+  }
+};
```

Comment:
Extract this code within `ProjectTypeIcon()` as a component. You can keep it within the same file.